platform.txt

```diff
@@ -215,3 +215,16 @@ tools.remoteproc_gen.script=run_arduino_gen.sh
 tools.remoteproc_gen.upload.params.verbose=
 tools.remoteproc_gen.upload.params.quiet=
 tools.remoteproc_gen.upload.pattern="{busybox}" sh "{path}/{script}" generate "{build.path}/{build.project_name}.elf" "{build.path}/run_arduino_{build.project_name}.sh"
+
+# Debugger configuration (general options)
+# ----------------------------------------
+# EXPERIMENTAL feature:
+# - this is alpha and may be subject to change without notice
+debug.executable={build.path}/{build.project_name}.elf
+debug.toolchain=gcc
+debug.toolchain.path={runtime.tools.xpack-arm-none-eabi-gcc-12.2.1-1.2.path}/bin
```

Comment:
I wonder if using `compiler.path` could works. It would avoid to forget to update gcc version when it is updated.
```suggestion
debug.toolchain.path={compiler.path}
```

Reply:
This works. Updated platform.txt with the simplification.